regulations/static/regulations/css/scss/partials/_reg_text.scss

```diff
@@ -5,6 +5,9 @@
 @import "../application_settings";
 
 .reg-text {
+    overflow-wrap: break-word;
+    overflow: auto;
+
```

Comment:
This was to correct long, unbroken strings that were making the reg text too wide.  Example:

https://regulations-pilot.cms.gov/42/460/Subpart-E/2021-05-12/#460-72-e

has a big URL in it that was causing issues